pom.xml

```diff
@@ -36,7 +36,7 @@
       <dependency>
         <groupId>com.amazonaws</groupId>
         <artifactId>aws-java-sdk-bom</artifactId>
-        <version>1.11.446</version>
+        <version>1.11.819</version>
```

Comment:
FYI, this will require a minor version bump IMO. We should also explicitly document our versioning policy while we're at it. For example: https://github.com/aws/aws-encryption-sdk-python/blob/master/VERSIONING.rst

Reply:
Added a VERSIONING file similar to aws-encryption-sdk-python